### PR TITLE
Fill in the body of a few tests

### PR DESCRIPTION
**Pull request by JordanVerasamy**
Also implements a deck specification parameter in `allocateHands`.

### CODE REVIEW

File: test/round.test.ts [hunk 1]
```diff
@@ -709,7 +742,96 @@ describe('RoundState', () => {
...
+                "AIR-Maneuver-3 (flipped)",
+                "SEA-Blockade-5",
+              ],
+              "TWO": Array [
+                "SEA-Maneuver-3 (flipped)",
+              ],
+            },
+          }
+        `);
+      });
```
Comment (keriwarr) [inline test/round.test.ts]: @JordanVerasamy this test is incorrect. blockade prevents cards from being played only in adjacent theaters.

Comment (keriwarr) [reply test/round.test.ts]: I can fix it if you don't feel like it :)

Comment (JordanVerasamy) [reply test/round.test.ts]: i got

File: test/round.test.ts [hunk 2]
```diff
@@ -248,9 +248,42 @@ describe('RoundState', () => {
         );
       });
 
-      it.todo("doesn't override blockade");
+      it.skip("doesn't override blockade", () => {
```
Comment (JordanVerasamy) [inline test/round.test.ts]: this test is also incorrect (blockade only comes into effect once there are 3 cards in the theater)

Comment (JordanVerasamy) [reply test/round.test.ts]: fix'd

**Comment by keriwarr** [comment]
Looks good, thanks!! :)

**Pull request merged**